README.md

```diff
@@ -44,7 +44,7 @@ $result = ArrayToXml::convert($array);
 After running this piece of code `$result` will contain:
 
 ```xml
-<?xml version="1.0"?>
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The library doesn't add `encoding` attribute by default. Output examples have to match the code samples provided.